src/Codeception/Application.php

```diff
@@ -80,7 +80,6 @@ protected function readCustomCommandsFromConfig(): void
      * Validate and get the name of the command
      *
      * @param CustomCommandInterface|string $commandClass
-     * @return string
      * @throws ConfigurationException
      */
     protected function getCustomCommandName($commandClass): string
```

Comment:
Based on the code and [documentation](https://codeception.com/docs/08-Customization#Custom-Commands) $commandClass is a string.